integration-tests/src/test/resources/testng.xml

```diff
@@ -1,11 +1,15 @@
+
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="ApiManager_IntegrationTest_Suite" verbose="1">
 
     <test name="REST_API_IntegrationTest" preserve-order="true" parallel="false">
         <packages>
             <!--commented to stop tests on build (need to uncomment later)-->
-            <!--<package name="org.wso2.carbon.apimgt.rest.integration.tests.*"/>-->
+            <package name="org.wso2.carbon.apimgt.rest.integration.tests.*"/>
```

Comment:
Can we comment this for now, please? since the deployment will not start an instance this would fail the build

Reply:
Hi Nuwan, 
Added.